app/controllers/sections/transportArranger/TransportArrangerIndexController.scala

```diff
@@ -40,7 +40,7 @@ class TransportArrangerIndexController @Inject()(
   def onPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedDataRequestWithUpToDateMovement(ern, arc) { implicit request =>
       if (TransportArrangerSection.isCompleted || TransportArrangerSection.needsReview) {
```

Comment:
We might want to change this (and other index routes) to be something more like this:

```scala
if(Section.isCompleted) {
  // CYA route
} else if(Section.needsReview) {
  // review route
} else {
  // section start
}
```

Reply:
Matt was going to deal with the .isCompleted stuff and going to CYA, so hence why I left it. He said it was easier to do it in his

Reply:
Yeah, already on it.